src/node_perf.cc

```diff
@@ -268,13 +267,14 @@ void MarkGarbageCollectionEnd(Isolate* isolate,
   // If no one is listening to gc performance entries, do not create them.
   if (!state->observers[NODE_PERFORMANCE_ENTRY_TYPE_GC])
     return;
-  GCPerformanceEntry* entry =
-      new GCPerformanceEntry(env,
-                             static_cast<PerformanceGCKind>(type),
-                             state->performance_last_gc_start_mark,
-                             PERFORMANCE_NOW());
-  env->SetUnrefImmediate(PerformanceGCCallback,
-                         entry);
+  auto entry = std::make_unique<GCPerformanceEntry>(
+      env,
+      static_cast<PerformanceGCKind>(type),
+      state->performance_last_gc_start_mark,
+      PERFORMANCE_NOW());
+  env->SetUnrefImmediate([entry = std::move(entry)](Environment* env) mutable {
```

Comment:
Why does this need `mutable`?

Reply:
Because here `entry` is moved again out of the function, and so it needs to be modifiable – Do you think a comment would help/be necessary here?